embedded-graphics/README.tpl

```diff
@@ -0,0 +1,82 @@
+<p align="center">
+    <img width="150" src="https://raw.githubusercontent.com/jamwaffles/embedded-graphics/191fe7f8a0fedc713f9722b9dc59208dacadee7e/assets/logo.svg?sanitize=true" alt="Embedded graphics logo">
+</p>
+<p align="center">
+    <a href="https://circleci.com/gh/jamwaffles/embedded-graphics/tree/master"><img src="https://circleci.com/gh/jamwaffles/embedded-graphics/tree/master.svg?style=shield" alt="Build Status"></a>
+    <a href="https://crates.io/crates/embedded-graphics"><img src="https://img.shields.io/crates/v/embedded-graphics.svg" alt="Crates.io"></a>
+    <a href="https://docs.rs/embedded-graphics"><img src="https://docs.rs/embedded-graphics/badge.svg" alt="Docs.rs"></a>
+    <a href="https://matrix.to/#/#rust-embedded-graphics:matrix.org"><img src="https://img.shields.io/matrix/rust-embedded-graphics:matrix.org" alt="embedded-graphics on Matrix"></a>
+</p>
+
+# Embedded graphics
+
+{{readme}}
+
+## Generating readmes
+
+The various `README.md` files in this project are generated from each crate's `lib.rs` comment. To
+regenerate a readme, ensure [`cargo-readme`](https://crates.io/crates/cargo-readme) is installed
```

Comment:
Isn't it possible to include cargo-readme in the dev-dependencies instead?

Reply:
Possibly, but then how would it run from a bash script? Can you install crates with binaries into `target/` or something?

Reply:
Right, I got confused for a bit, sorry about that ^^

Reply:
No problem! The build scripts/environment is pretty involved for e-g :grimacing: